schemas/io.snowplow.foundation/object/jsonschema/1-0-0

```diff
@@ -0,0 +1,36 @@
+{
+  "description": "Schema for a generic object entity",
+  "properties": {
+    "name": {
+      "type": "string",
+      "maxLength": 255,
+      "description": "The name of the object or property"
+    },
+    "svalue": {
+      "type": "string",
+      "maxLength": 4096,
+      "description": "The object or property value, if it is a string"
+    },
+    "nvalue": {
+      "type": "number",
+      "description": "The object or property value, if it is a number"
+    },
+    "tvalue": {
+      "type": "string",
+      "format": "date-time",
+      "description": "The object or property value, if it is a timestamp"
+    }
+  },
+  "additionalProperties": false,
+  "type": "object",
+  "required": [
+    "name"
+  ],
+  "self": {
+    "vendor": "io.snowplow.foundation",
+    "name": "object",
+    "format": "jsonschema",
+    "version": "1-0-0"
+  },
```

Comment:
This is a weird old schema. Not something we'd ever suggest someone uses in the real world as its so generic. Whats the use case for a schema like this?

Reply:
Good point, I just checked and it's not actually used in any of the tutorials, so I'm happy to ditch this one